wsd/wopi/WopiStorage.cpp

```diff
@@ -237,6 +237,11 @@ WopiStorage::WOPIFileInfo::WOPIFileInfo(const FileInfo& fileInfo, Poco::JSON::Ob
     JsonUtil::findJSONValue(object, "UserExtraInfo", _userExtraInfo);
     JsonUtil::findJSONValue(object, "UserPrivateInfo", _userPrivateInfo);
     JsonUtil::findJSONValue(object, "ServerPrivateInfo", _serverPrivateInfo);
+    if (auto settingsJSON = object->getObject("UserSettings"))
+        JsonUtil::findJSONValue(settingsJSON, "uri", _userSettingsUri);
+
+    JsonUtil::findJSONValue(object, "UserSettingsUri", _userSettingsUri);
```

Comment:
We are overwriting `_userSettingsUri` ? Doesn't look right. 

Is the idea to use `UserSettings.uri` if available, but always prefer `UserSettingsUri`, if set ?

A comment is warranted, if this is indeed intentional.

Reply:
I added that for testing and forgot to remove : ) 

fixed : https://github.com/CollaboraOnline/online/pull/10933/commits/438500f319e3969c16a16dd04bad6ba2187cda65. Thanks for pointing out :)